cmd/ui/src/views/Explore/ExploreSearch/CypherSearch.tsx

```diff
@@ -54,11 +54,15 @@ const useStyles = makeStyles((theme) => ({
         borderColor: 'rgba(0,0,0,.23)',
         borderRadius: theme.shape.borderRadius,
         backgroundColor: 'white',
-        paddingTop: '5px',
-        minHeight: '120px',
-        // enables drag n drop resizing of editor
-        resize: 'vertical',
-        maxHeight: '500px',
+        minHeight: '94px',
+        maxHeight: '94px',
+        overflow: 'scroll',
```

Comment:
Not sure if its me, but I tested in chrome, and with overflow: 'scroll' it seems to be showing empty scrollbars even if no need to scroll (See screenshots below). But not in safari. If you change it to overflow: 'auto' it doesnt happen.

<img width="1799" alt="Screenshot 2024-09-10 at 3 20 12 PM" src="https://github.com/user-attachments/assets/067f59ce-0fc2-484c-8470-d0b4e2928bc7">
<img width="1856" alt="Screenshot 2024-09-10 at 3 21 23 PM" src="https://github.com/user-attachments/assets/93004437-d451-4266-bf82-05c3e44d3d6f">